gradle-plugin/it/src/it/dockerfile-simple/expected/openshift.yml

```diff
@@ -58,9 +58,9 @@ items:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-          image: gradle/dockerfile-simple:latest
+          image: "@endsWith('dockerfile-simple:latest')@"
```

Comment:
What's the complete name? can we keep it specific or is it not stable enough?

Reply:
Complete name is resolved as namespace if I'm logged into an OpenShift cluster. For my case, it gets resolved to `rokumar-dev`

Reply:
So probably something needs to be tuned in the build.gradle configuration so no real cluster is checked during the test.

Reply:
https://github.com/eclipse/jkube/blob/72529d1aa599e5065452850c996a1e1fe777da46/gradle-plugin/it/src/it/dockerfile-simple/build.gradle#L15

openshift equivalent should fix it, right?

Reply:
I'm trying but it doesn't seem to work. Looks like `.kube/config` file is read and namespace is getting picked from there:

https://github.com/eclipse/jkube/blob/f6cd5409d0e8a0e4cb2fb44ea1bd359c5c8489e9/jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/KubernetesHelper.java#L433-L435

Reply:
OK, we can try to solve this later.